python/ml4ir/applications/ranking/tests/test_ranklib_in_ml4ir.py

```diff
@@ -35,7 +41,8 @@ def test_ranklib_in_ml4ir(self):
         """Creates a relevance dataset using ranklib format. Labels are graded relevance"""
 
         io = local_io.LocalIO()
-        exFeatureConfig = self.parse_config(TFRecordTypeKey.SEQUENCE_EXAMPLE, self.feature_config_yaml, io)
+        exFeatureConfig = self.parse_config(
```

Comment:
is this from an automatic tool you use? yapf or another?

Reply:
I have "AutoPEP8" plugin on my SublimeText3. It is pretty lightweight. I like it.